.travis.yml

```diff
@@ -1,22 +1,43 @@
 language: cpp
 
-os:
```

Comment:
Did you intentionally add .travis.yml changes to this pull request? (I am in favour of those changes, just asking.)